hal/lib/athena/FPGAEncoder.cpp

```diff
@@ -34,17 +34,17 @@ HAL_FPGAEncoderHandle HAL_InitializeFPGAEncoder(
     HAL_Handle digitalSourceHandleB, HAL_AnalogTriggerType analogTriggerTypeB,
     HAL_Bool reverseDirection, int32_t* index, int32_t* status) {
   bool routingAnalogTriggerA = false;
-  uint8_t routingPinA = 0;
+  uint8_t routingChannelA = 0;
   uint8_t routingModuleA = 0;
-  bool successA =
-      remapDigitalSource(digitalSourceHandleA, analogTriggerTypeA, routingPinA,
-                         routingModuleA, routingAnalogTriggerA);
+  bool successA = remapDigitalSource(digitalSourceHandleA, analogTriggerTypeA,
+                                     routingChannelA, routingModuleA,
+                                     routingAnalogTriggerA);
```

Comment:
This patch originally changed everything to use the word "pin". When I changed it to "channel", some name and formatting artifacts were left behind, and I didn't catch all of them. However, the formatting on this line looks fine; "routingPinA" became "routingChannelA". Moving everything onto one line and running the formatter produces this output.